libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -1428,16 +1428,27 @@ class RunnableMap(RunnableSerializable[Input, Dict[str, Any]]):
 
     def __init__(
         self,
-        steps: Mapping[
-            str,
-            Union[
-                Runnable[Input, Any],
-                Callable[[Input], Any],
-                Mapping[str, Union[Runnable[Input, Any], Callable[[Input], Any]]],
-            ],
+        __steps: Optional[
+            Mapping[
+                str,
+                Union[
+                    Runnable[Input, Any],
+                    Callable[[Input], Any],
+                    Mapping[str, Union[Runnable[Input, Any], Callable[[Input], Any]]],
+                ],
```

Comment:
for readability maybe worth aliasing

Reply:
I dont think aliases work with type variables in python correctly